pkg/kp/rcstore/consul_store.go

```diff
@@ -220,6 +220,16 @@ func (s *consulStore) SetDesiredReplicas(id fields.ID, n int) error {
 	})
 }
 
+func (s *consulStore) MutateDesiredReplicas(id fields.ID, n int) error {
```

Comment:
what about "Add" instead of "Mutate". I thought the second argument was the desired value rather than an operand